README.md

```diff
@@ -6,7 +6,7 @@
 | --- | --- | --- | --- |
 | [Caffe2](http://caffe2.ai) | [onnx/onnx-caffe2](https://github.com/onnx/onnx-caffe2) | [Exporting](tutorials/Caffe2OnnxExport.ipynb) | [Importing](tutorials/OnnxCaffe2Import.ipynb) |
 | [PyTorch](http://pytorch.org/) | [part of pytorch package](http://pytorch.org/docs/master/onnx.html) | [Exporting](tutorials/PytorchOnnxExport.ipynb), [Extending support](tutorials/PytorchAddExportSupport.md) | coming soon |
-| [CNTK](https://github.com/Microsoft/CNTK) | coming soon | coming soon | coming soon |
+| [CNTK](https://github.com/Microsoft/CNTK) | coming soon | [Exporting](tutorials/CntkOnnxExport.ipynb) | [Importing](tutorials/OnnxCntkImport.ipynb) |
```

Comment:
The installation cell should say its part of CNTK and link to the CNTK installation instructions